### ISSUE TITLE
azurerm_container_app_environment logs destination option is missing

### ISSUE THREAD

**Issue by satyakrish** [hashicorp/terraform-provider-azurerm#22777]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Terraform Version

1.3.9

### AzureRM Provider Version

3.67

### Affected Resource(s)/Data Source(s)

azurerm_container_app_environment

### Terraform Configuration Files

```hcl
resource "azurerm_container_app_environment" "example" {
  name                       = "my-environment"
  location                   = azurerm_resource_group.example.location
  resource_group_name        = azurerm_resource_group.example.name
  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
}
```

### Debug Output/Panic Output

```shell
n/a
```

### Expected Behaviour

azurerm_container_app_environment shoudl support logs destination parameter and allow users to configure azure-monitor, log-analytics, none as options.

Currently both portal and az cli supports this option.

### Actual Behaviour

logs destination parameter  is not available

**Comment by jhutchings** [comment]
Yeah in looking at this the default declaration of the Managed Environment doesn't provide the [AppLogsConfiguration to the Properties](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/containerapps/container_app_environment_resource.go#L159-L166) and the only reference to the Properties.AppLogsConfiguration happens [when you define a Logs Analytics Workspace Id](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/containerapps/container_app_environment_resource.go#L168-L201)

**Comment by asbtho-eq** [comment]
Any update on this?

**Comment by katbyte** [comment]
Good news, `logs_destination` on `azurerm_container_app_environment` is available per the [documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/container_app_environment), as such we are going to close this request out as completed.

If this does not cover your use case on the latest version of the provider (v5.x) please let us know with a comment or open a new issue referencing this one. Thank you!
